### PR TITLE
dashboard metrics

### PR DESCRIPTION
**Pull request by prakashsvmx**
image

Note: This PR depends on https://github.com/minio/minio/pull/17663

Note: `Buckets`  i.e: Number of Buckets  requires  update to new endpoint may be, as  it is not available at cluster level `minio_cluster_usage_total_bytes` does not publish bucket name.

@harshavardhana

### CODE REVIEW

**Comment by harshavardhana** [comment]
> Note: `Buckets` i.e: Number of Buckets requires update to new endpoint may be, as it is not available at cluster level `minio_cluster_usage_total_bytes` does not publish bucket name.

will add this as `minio_cluster_total_buckets` @prakashsvmx

**Comment by prakashsvmx** [comment]
Thank you @harshavardhana . i have updated this PR to use the `minio_cluster_buckets_total` and the screenshot for reference.

File: restapi/admin_info.go [hunk 1]
```diff
@@ -305,7 +305,7 @@ var widgets = []Metric{
 		},
 		Targets: []Target{
 			{
-				Expr:         `sum(minio_bucket_usage_total_bytes{$__query}) by (instance)`,
+				Expr:         `sum(minio_cluster_usage_total_bytes{$__query}) by (instance)`,
```
Comment (harshavardhana) [inline restapi/admin_info.go:308]: This is not a sum anymore @prakashsvmx since it's already a sum()

File: restapi/admin_info.go [hunk 2]
```diff
@@ -598,7 +599,7 @@ var widgets = []Metric{
 		},
 		Targets: []Target{
 			{
-				Expr:         `topk(1, sum(minio_bucket_usage_object_total{$__query}) by (instance))`,
+				Expr:         `topk(1, sum(minio_cluster_usage_object_total{$__query}) by (instance))`,
```
Comment (harshavardhana) [inline restapi/admin_info.go:602]:
```suggestion
				Expr:         `topk(1, minio_cluster_usage_object_total{$__query} by (instance))`,
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: restapi/admin_info.go (hunk 1 above)
Comment (prakashsvmx) [reply restapi/admin_info.go]: Thank you, i have updated the query and verified.

Comment (prakashsvmx) [reply restapi/admin_info.go]:
```
# TYPE minio_cluster_bucket_total gauge
minio_cluster_bucket_total{server="127.0.0.1:22000"} 3
minio_cluster_usage_object_total{server="127.0.0.1:22000"} 12
minio_cluster_usage_total_bytes{server="127.0.0.1:22000"} 1.09520916e+08

# TYPE minio_cluster_objects_size_distribution gauge
minio_cluster_objects_size_distribution{range="BETWEEN_1024_B_AND_1_MB",server="127.0.0.1:22000"} 1
minio_cluster_objects_size_distribution{range="BETWEEN_10_MB_AND_64_MB",server="127.0.0.1:22000"} 0
minio_cluster_objects_size_distribution{range="BETWEEN_128_MB_AND_512_MB",server="127.0.0.1:22000"} 0
minio_cluster_objects_size_distribution{range="BETWEEN_1_MB_AND_10_MB",server="127.0.0.1:22000"} 1
minio_cluster_objects_size_distribution{range="BETWEEN_64_MB_AND_128_MB",server="127.0.0.1:22000"} 1
minio_cluster_objects_size_distribution{range="GREATER_THAN_512_MB",server="127.0.0.1:22000"} 0
minio_cluster_objects_size_distribution{range="LESS_THAN_1024_B",server="127.0.0.1:22000"} 9

```

File: restapi/admin_info.go [hunk 3]
```diff
@@ -331,7 +331,7 @@ var widgets = []Metric{
 		},
 		Targets: []Target{
 			{
-				Expr:         `max by (range) (minio_bucket_objects_size_distribution{$__query})`,
+				Expr:         `max by (range) (minio_cluster_objects_size_distribution{$__query})`,
```
Comment (harshavardhana) [inline restapi/admin_info.go:334]:
```suggestion
				Expr:         `minio_cluster_objects_size_distribution{$__query}`,
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]
one more change `max by range` is not needed apparently.

**Comment by harshavardhana** [comment]
https://github.com/minio/minio/pull/17683 PTAL

File: restapi/admin_info.go (hunk 3 above)
Comment (prakashsvmx) [reply restapi/admin_info.go]: Thank you again. i have updated the query. also updated the widget as per new response structure.

**Review by harshavardhana — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**